config/deploy/templates/puma.rb.erb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env puma
+
+directory '<%= current_path %>'
```

Comment:
As I can see it is common practice.. https://docs.omniref.com/ruby/gems/capistrano3-puma/0.4.1/files/lib/capistrano/templates/puma.rb.erb or http://www.rubydoc.info/gems/capistrano-puma-simple.